src/Select.tsx

```diff
@@ -294,30 +302,20 @@ const Select = React.forwardRef(
         return [];
       }
 
+      // if (values)
+
       return values;
     }, [internalValue, convert2LabelValues, mode]);
 
     // Fill label with cache to avoid option remove
     const [mergedValues, getMixedOption] = useCache(rawLabeledValues, valueOptions);
 
     const displayValues = React.useMemo(() => {
-      // `null` need show as placeholder instead
```

Comment:
Since when `null` as value will  returns [], this logic code is no looger needed. When `value={null}`, the placeholder will show up correctly.
因为 null 作为空值已经做了返回 [] 的处理，所以这段特殊处理此 issue 的逻辑已经不需要了，`value={null}` 时 placeholder 会正确显示。